### ISSUE TITLE
Add ActivityId and RelatedActivityId to the events

### ISSUE THREAD

**Issue by markantill** [Microsoft/krabsetw#64]
ActivityId and RelatedActivityId header fields aren't available through the API but were available in the old TraceEvent library are useful for correlating events.

**Comment by swannman** [comment]
Good catch Mark, that would be useful. If you’d be willing to submit a pull request that adds these, we’ll review and merge.

**Comment by pathtofile** [comment]
Put up a PR for ActivityID: https://github.com/microsoft/krabsetw/pull/129
Didn't touch RelatedActivityID, that's a deeper whole involving parsing extended data items

**Comment by swannman** [comment]
@pathtofile thank you! Merged #129.
